### ISSUE TITLE
build error

### ISSUE THREAD

**Issue by amorenew** [nisrulz/easydeviceinfo#3]
hi
i can't build check this error
Error:Execution failed for task ':app:processDebugResources'.
> Error: more than one library with package name 'com.google.android.gms'
in my gradle

 compile 'com.google.android.gms:play-services-maps:7.8.0'

**Comment by nisrulz** [comment]
EasyDeviceInfo library under the hood  [references](https://github.com/nisrulz/easydeviceinfo/blob/master/library/build.gradle#L50) the google play services ad dependency, to enable retrieval of Advertiser ID( only on request via a function, not on its own)

Since you have an older version of google play services hence the issue. If you upgrade to latest version `8.4.0` it should work.

If you donot want to upgrade then you will have to instruct gradle to resolve to the version you are referring i.e `7.4.0`

basically 

```gradle
configurations.all {
  resolutionStrategy.force 'com.google.android.gms:play-services:7.8.0'
}

```
I wrote a blog post recently about how to resolve dependency conflicts, you can check [here](http://crushingcode.co/the-curious-case-of-dependency-conflicts/) for more information.

**Comment by amorenew** [comment]
thanks for help
